README.md

```diff
@@ -0,0 +1,38 @@
+# Provider
+Provider streams the iOS devices for controlfloor. This bridges the phone to the browser.
+
+# Basic Install Instructions
+* Clone repos
+git clone ios_remote_provider
+git clone controlfloor
+git clone ios_video_app
+
+* Build nng - https://github.com/nanomsg/nng
+cmake, make, make install
+
+* Build repos
+
+cd controlfloor
+make
+./main run
+
+Run https://yourip:8080 to see if controlfloor is running
```

Comment:
If defaulting to https it should be mentioned in the readme to run the script to regenerate the self signed certificate in case it expires.